EIPS/eip-3475.md

```diff
@@ -1,448 +1,418 @@
 ```
 
 ### EVENT
 
-`"eventIssueBond"` MUST trigger when Bonds are issued. This SHOULD not include zero value Issuing. 
+`"Issue"` MUST trigger when Bonds are issued. This SHOULD not include zero value Issuing.
+
 
 e.g. 
 
 `"issue by address(operator) 500 DEBIT-USD Bond(Nonce14) to 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef"`
 
 ```
-event eventIssueBond(address _operator, address _to, uint256 class, uint256 nonce, uint256 _amount); 
+event Issue(address indexed _operator, address indexed _to, uint256 classId, uint256 nonceId, uint256 amount); 
 ```
 
-`"eventRedeemBond"` MUST trigger when Bonds are redeemed. This SHOULD not include zero value redemption. When burn a bond MUST not create this event(Use`"eventBurnBond" ` instead).
+`"Redeem"` MUST trigger when Bonds are redeemed. This SHOULD not include zero value redemption.
 
 e.g. 
 
 `"redeem by address(_operator) 500 DEBIT-USD Bond(Nonce14) to 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef"`
 
 ```
-event eventRedeemBond(address _operator, address _from, uint256 class, uint256 nonce, uint256 _amount);
+event Redeem(address indexed _operator, address indexed _from, uint256 classId, uint256 nonceId, uint256 amount);
 ```
 
-`"eventBurnBond"` MUST trigger when Bonds are burned. This SHOULD not include zero value burning. When redeem a bond MUST not create this event(Use`"event redeemBond" ` instead).
+`"Burn"` MUST trigger when Bonds are burned. This SHOULD not include zero value burning.
 
 e.g. 
 
 `"burn by address(_operator) 500 DEBIT-USD Bond(Nonce14) from 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef"`
 
 ```
-event eventBurnBond(address _operator, address _from, uint256 class, uint256 nonce, uint256 _amount);
+event Burn(address indexed _operator, address indexed _from, uint256 classId, uint256 nonceId, uint256 amount);
 ```
 
-`"eventTransferBond"` MUST trigger when Bonds are transferred. This SHOULD not include zero value transfers. Transfer event with the `_from` `0x0` MUST not create this event(Use`"event issueBond" `instead ). Transfer event with the `_to` `0x0` MUST not create this event(Use`"event redeemBond" `when redemption, and `"event burnBond" `when burning).
+`"Transfer"` MUST trigger when Bonds are transferred. This SHOULD not include zero value transfers. Transfer event with the `_from` `0x0` MUST not create this event(Use`"event Issued" `instead ). Transfer event with the `_to` `0x0` MUST not create this event(Use`"event Redeemed" `when redemption, and `"event Burned" `when burning).
 
 e.g.
 
 `"transfer by address(_operator) 500 DEBIT-USD Bond(Nonce14) from 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef, to address(_to)"`
 
 ```
-event eventTransferBond(address _operator, address _from, address _to, uint256 class, uint256 nonce, uint256 _amount);
+event Transfer(address indexed _operator, address indexed _from, address indexed _to, uint256 classId, uint256 nonceId, uint256 amount);
 ```
 
 ## Backwards Compatibility
```

Comment:
Ahh, I'm so sorry! I missed that this EIP doesn't include a [Rationale](https://eips.ethereum.org/EIPS/eip-1#what-belongs-in-a-successful-eip) section, which is mandatory.